### DISCUSSION TITLE
Copilot chat does not seem to work on VSCodium

### DISCUSSION THREAD

**Discussion by m1212e** [community/community#101665] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi, it seems like the Copilot Chat extension is not working on VSCodium out of the box.

![image](https://github.com/community/community/assets/14091540/bd463fd9-d257-45b8-9f20-91dd68109a55)

This is unfortunate, since it is a paid service and there definitely is demand as far as I can tell from forum posts, etc.
Is the team working on this or do I just have to look for another service?

Thanks!

**Comment by guillermoarellano** [comment]
This is terrible news. I was trying to use copilot on my machine with the VSCodium and I got an error message when trying to download the plugin that it only works with Visual Studio Code. I guess Micro$oft only wants the plugin to work with their IDE for tracking purposes. Bummer.

**Comment by resimard** [comment]
I noticed this as well, but the extension installed just fine.  When I open the chat, I notice that there is no prompt, and I cannot send messages.  I'm happy to have the suggestions working, but I'd also like to have the chat available for my monthly payment.

![image](https://github.com/community/community/assets/11303079/cd7dd3a6-8085-401e-a914-12e0149521f3)

I am using VSCodium, having migrated from Sublime Text which had a sub-optimal implementation of Copilot.  Hopefully there is some support for the chat coming on VSCodium, since I run Linux and I'm not interested in having Microsoft products (VSCode) on my laptop :)

**Comment by Equilibrium94** [comment]
I get the same error even on 1.95.3. sad
